src/validation.cpp

```diff
@@ -158,6 +158,8 @@ class CChainState {
     std::multimap<CBlockIndex*, CBlockIndex*> mapBlocksUnlinked;
     CBlockIndex *pindexBestInvalid = nullptr;
 
+    std::unique_ptr< CCheckQueue<CScriptCheck> >scriptcheckqueue;
```

Comment:
In Bitcoin, dose not change this and Line#1822-1824. This change necessary?

Reply:
Yes, bitcoin master does not contain this change. The same issue is reported in 25448 and fixed in #26762. These are ported from that PR.